frontend/src/features/books/BookList.jsx

```diff
@@ -52,7 +52,8 @@ return (
               <td style={{ padding: '10px' }}>{book.title}</td>
               <td style={{ padding: '10px' }}>{book.author}</td>
               <td style={{ padding: '10px' }}>{book.description}</td>
-              <td><Link to={`/books/${book.id}`}>View</Link></td>
+              <td><Link to={`/books/${book.id}`}>View  |</Link></td>
+              <td><Link to={`/books/${book.id}/edit`}> git chcekout Edit</Link></td>
```

Comment:
@nshelokar-vf, `chcekout` is this intentional?
Please use proper naming conventions here

Reply:
It was by mistake.